org.civicrm.reqangular/src/common/bundles/angularDate.js

```diff
@@ -0,0 +1,5 @@
+define([
+    'common/services/angularDate/date-format'
```

Comment:
Also, for file names (and therefore module names): please use dasherized style (`angular-date`)